Reboot command is working again

lzap opened the pull request:
README has been rewritten, some other improvements like logging to pty1
console, nice banner and minimize improvements.

Releasing new images right now, uploading to my site:

http://lzap.fedorapeople.org/zzz/

lzap committed:
Reboot command is working again

README has been rewritten, some other improvements like logging to pty1
console, nice banner and minimize improvements

lzap commented:
![asciart](https://f.cloud.github.com/assets/49752/1774849/085962ea-67ff-11e3-98c4-ccd2ffb2f411.png)

lzap commented:
Oh damn I was too slow taking the screenshot. Interesting but I see now a bug @GregSutcliffe ^^^ Facter failed to parse ipaddress during this boot. Strange.

lzap commented:
![screenshot](https://f.cloud.github.com/assets/49752/1774866/49b442dc-67ff-11e3-9012-b2eba01900d3.png)

lzap commented:
Btw worked after reboot, had to be some issue with network, but we start the discover host service after network is up. Strange.

GregSutcliffe commented on extra/ovirt-node-plugin-foreman/ovirt-node-plugin-foreman.spec.in:
```diff
@@ -117,5 +128,9 @@ sed -i '/\[Service\]/a Environment="FACTERLIB=/usr/share/ovirt-node-plugin-forem
 %{_sysconfdir}/ovirt-plugins.d
 
 %changelog
+* Wed Oct 30 2013 Lukas Zapletal <lzap+rpm@redhat.com> 0.2.0-1
```
Cutnpaste error, should be 18th Dec :)

GregSutcliffe commented:
@lzap both the macaddress and ipaddress facts depend on "ifconfig -a" so I'm surprised it worked without...

lzap commented:
Cool I have rebased and fixed the log, also I have added one extra line:

```
systemctl enable NetworkManager-wait-online.service
```

which should make nm to wait until an IP address is discovered. This should prevent the error when IP is not yet ready.

GregSutcliffe commented:
Merged, thnks @lzap!
